src/Controller/CreateCaseController.php

```diff
@@ -16,7 +17,7 @@ class CreateCaseController extends AbstractController
     /**
      * @Route("/municipality/{municipality_name}/board/{board_name}/case/create", name="rescase")
      */
-    public function createCase(BoardRepository $boardRepository, MunicipalityRepository $municipalityRepository, Request $request, string $municipality_name, string $board_name): Response
+    public function createCase(BoardRepository $boardRepository, CaseManager $caseManager, MunicipalityRepository $municipalityRepository, Request $request, string $municipality_name, string $board_name): Response
```

Comment:
It strikes me now that there is a lot of non-camelcase arguments here. We should probably rename them to follow that standard.

Reply:
Renamed arguments to follow naming convention.